lib/compat/wordpress-6.7/post-formats.php

```diff
@@ -0,0 +1,23 @@
+<?php
+/**
+ * Temporary compatibility shims for changes to post formats in Gutenberg.
```

Comment:
"Temporary" generally means "permanent" in software unless there are clear conditions listed under which someone can change it. What makes this temporary? What needs to happen to remove it? How should someone remove it when that occurs?

Is this more or less saying this?

```php
/**
 * Shims for changes to post formats until they are merged in all
 * versions of WordPress which Gutenberg supports.
 *
 * @package gutenberg
 */
```

Reply:
As I understand it, it is temporary in the plugin because the compatibility files are removed as soon as possible:

- The change is added to core.
- The plugin has a minimum required WordPress version where the change is implemented.

Reply:
I think this change is not relevant unless you intended to update all the file headers in lib/compat/version.

Reply:
"Temporary" can be very confusing though. I don't think we have to change existing code to address it. What you wrote in the first reply to my comment would be considerably more helpful to someone who comes around to this code later and wonders why it's still there.